assessment/models.py

```diff
@@ -48,10 +49,18 @@ def get_tool_data(self) -> dict:
         }
         return tool_base_data
 
-    def get_end_working_time(self, start_time: datetime.time):
-        temporary_datetime = datetime.datetime(2000, 1, 1, start_time.hour, start_time.minute, start_time.second)
-        temporary_datetime = temporary_datetime + datetime.timedelta(minutes=self.duration_in_minutes)
-        return temporary_datetime.time()
+    def get_end_working_time_if_executed_on_event_date(self, start_time: datetime.time, event_date):
```

Comment:
Hello jo, if I might ask, does this mean that the end working time will be sent to the frontend in datetime format? If so, let me know!

Reply:
Hello there, Rashad. Yes, the end working time in datetime format in UTC will be sent to the front end. I'm hoping the time conversion won't be too difficult :).

Reply:
Okay thanks jo! This would actually help massively in the frontend, as it will make our assignment functionality more robust